tests/test_run_config_result_comparator.py

```diff
@@ -126,8 +139,50 @@ def test_equal_weight_multi(self):
             model_name="test_model",
             model_config_names=["test_model_config0", "test_model_config1"])
 
+    def test_unequal_weight_multi(self):
```

Comment:
I see a test that confirms the input format (yaml only).
I see a test that confirms the weights work properly.
I don't see anything that confirms that we go properly from yaml->comparison objects->correct comparison